### ISSUE TITLE
blosc tests fail in windows 11

### ISSUE THREAD

**Issue by bjarthur** [stevengj/Blosc.jl#90]
```
(@v1.9) pkg> test Blosc
     Testing Blosc
      Status `C:\Users\arthurb\AppData\Local\Temp\jl_kMcaTe\Project.toml`
  [a74b3585] Blosc v0.7.3
  [0b7ba130] Blosc_jll v1.21.4+0
  [8dfed614] Test `@stdlib/Test`
      Status `C:\Users\arthurb\AppData\Local\Temp\jl_kMcaTe\Manifest.toml`
  [a74b3585] Blosc v0.7.3
  [692b3bcd] JLLWrappers v1.5.0
  [21216c6a] Preferences v1.4.1
  [0b7ba130] Blosc_jll v1.21.4+0
  [5ced341a] Lz4_jll v1.9.4+0
  [3161d3a3] Zstd_jll v1.5.5+0
  [56f22d72] Artifacts `@stdlib/Artifacts`
  [2a0f44e3] Base64 `@stdlib/Base64`
  [ade2ca70] Dates `@stdlib/Dates`
  [b77e0a4c] InteractiveUtils `@stdlib/InteractiveUtils`
  [8f399da3] Libdl `@stdlib/Libdl`
  [56ddb016] Logging `@stdlib/Logging`
  [d6f4376e] Markdown `@stdlib/Markdown`
  [de0858da] Printf `@stdlib/Printf`
  [9a3f8284] Random `@stdlib/Random`
  [ea8e919c] SHA v0.7.0 `@stdlib/SHA`
  [9e88b42a] Serialization `@stdlib/Serialization`
  [fa267f1f] TOML v1.0.3 `@stdlib/TOML`
  [8dfed614] Test `@stdlib/Test`
  [4ec0a83e] Unicode `@stdlib/Unicode`
  [83775a58] Zlib_jll v1.2.13+0 `@stdlib/Zlib_jll`
Precompiling project...
  6 dependencies successfully precompiled in 11 seconds. 1 already precompiled.
     Testing Running tests...

Invalid instruction at 000000006800d83e: 0xc5, 0xfa, 0x7e, 0x54, 0x24, 0x28, 0x48, 0x89, 0x54, 0x24, 0x28, 0xc5, 0xfa, 0x7e, 0x5c

Please submit a bug report with steps to reproduce this fault, and any error messages that follow (in their entirety). Thanks.
Exception: EXCEPTION_ILLEGAL_INSTRUCTION at 0x6800d83e -- .text at C:\Users\arthurb\.julia\artifacts\e48a0bacd2ce6a108c7edc024073f889d920d522\bin\libblosc.dll (unknown line)
in expression starting at C:\Users\arthurb\.julia\packages\Blosc\jk4Np\test\runtests.jl:27
.text at C:\Users\arthurb\.julia\artifacts\e48a0bacd2ce6a108c7edc024073f889d920d522\bin\libblosc.dll (unknown line)
pthread_once at /workspace/srcdir/mingw-w64-v7.0.0/mingw-w64-libraries/winpthreads/src\thread.c:745
blosc_internal_shuffle at C:\Users\arthurb\.julia\artifacts\e48a0bacd2ce6a108c7edc024073f889d920d522\bin\libblosc.dll (unknown line)
blosc_c at C:\Users\arthurb\.julia\artifacts\e48a0bacd2ce6a108c7edc024073f889d920d522\bin\libblosc.dll (unknown line)
do_job at C:\Users\arthurb\.julia\artifacts\e48a0bacd2ce6a108c7edc024073f889d920d522\bin\libblosc.dll (unknown line)
blosc_compress at C:\Users\arthurb\.julia\artifacts\e48a0bacd2ce6a108c7edc024073f889d920d522\bin\libblosc.dll (unknown line)
blosc_compress at C:\Users\arthurb\.julia\packages\Blosc\jk4Np\src\Blosc.jl:36 [inlined]
#compress!#3 at C:\Users\arthurb\.julia\packages\Blosc\jk4Np\src\Blosc.jl:96
compress! at C:\Users\arthurb\.julia\packages\Blosc\jk4Np\src\Blosc.jl:74 [inlined]
#compress#6 at C:\Users\arthurb\.julia\packages\Blosc\jk4Np\src\Blosc.jl:111
compress at C:\Users\arthurb\.julia\packages\Blosc\jk4Np\src\Blosc.jl:109 [inlined]
#compress#7 at C:\Users\arthurb\.julia\packages\Blosc\jk4Np\src\Blosc.jl:117 [inlined]
compress at C:\Users\arthurb\.julia\packages\Blosc\jk4Np\src\Blosc.jl:115
unknown function (ip: 0000029e1d7163e6)
jl_apply at C:/workdir/src\julia.h:1880 [inlined]
do_call at C:/workdir/src\interpreter.c:126
eval_value at C:/workdir/src\interpreter.c:226
eval_stmt_value at C:/workdir/src\interpreter.c:177 [inlined]
eval_body at C:/workdir/src\interpreter.c:624
eval_body at C:/workdir/src\interpreter.c:533
jl_interpret_toplevel_thunk at C:/workdir/src\interpreter.c:762
jl_toplevel_eval_flex at C:/workdir/src\toplevel.c:912
jl_toplevel_eval_flex at C:/workdir/src\toplevel.c:856
ijl_toplevel_eval at C:/workdir/src\toplevel.c:921 [inlined]
ijl_toplevel_eval_in at C:/workdir/src\toplevel.c:971
eval at .\boot.jl:370 [inlined]
include_string at .\loading.jl:1903
_include at .\loading.jl:1963
include at .\client.jl:478
unknown function (ip: 0000029e1d710086)
jl_apply at C:/workdir/src\julia.h:1880 [inlined]
do_call at C:/workdir/src\interpreter.c:126
eval_value at C:/workdir/src\interpreter.c:226
eval_stmt_value at C:/workdir/src\interpreter.c:177 [inlined]
eval_body at C:/workdir/src\interpreter.c:624
jl_interpret_toplevel_thunk at C:/workdir/src\interpreter.c:762
jl_toplevel_eval_flex at C:/workdir/src\toplevel.c:912
jl_toplevel_eval_flex at C:/workdir/src\toplevel.c:856
ijl_toplevel_eval at C:/workdir/src\toplevel.c:921 [inlined]
ijl_toplevel_eval_in at C:/workdir/src\toplevel.c:971
eval at .\boot.jl:370 [inlined]
exec_options at .\client.jl:280
_start at .\client.jl:522
jfptr__start_29544 at C:\Users\arthurb\.julia\juliaup\julia-1.9.3+0.x64.w64.mingw32\lib\julia\sys.dll (unknown line)
jl_apply at C:/workdir/src\julia.h:1880 [inlined]
true_main at C:/workdir/src\jlapi.c:573
jl_repl_entrypoint at C:/workdir/src\jlapi.c:717
mainCRTStartup at C:/workdir/cli\loader_exe.c:59
BaseThreadInitThunk at C:\WINDOWS\System32\KERNEL32.DLL (unknown line)
RtlUserThreadStart at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
Allocations: 2998 (Pool: 2986; Big: 12); GC: 0
ERROR: Package Blosc errored during testing
```

**Comment by mkitti** [comment]
I think this is probably https://github.com/Blosc/c-blosc/issues/371 and a bug with Blosc_jll.

This should have been resolved in Blosc v1.21.5 after https://github.com/Blosc/c-blosc/pull/373

This is an issue with https://github.com/JuliaBinaryWrappers/Blosc_jll.jl and more specifically https://github.com/JuliaPackaging/Yggdrasil/blob/5d74dc4091d5a1590e20ebc45c92687c239c2f43/B/Blosc/build_tarballs.jl

**Comment by bjarthur** [comment]
thanks @mkitti.  `] add Blosc_jll@1.21.2` fixed it for me.
